python/pip_install/pip_repository.bzl

```diff
@@ -12,7 +12,7 @@ def _construct_pypath(rctx):
         rctx: Handle to the repository_context.
     Returns: String of the PYTHONPATH.
     """
-    rctx.file("BUILD", "")
+    rctx.file("BUILD.bazel", "")
```

Comment:
Can you move this line inside the else block [here](https://github.com/bazelbuild/rules_python/pull/457/files#diff-bced444149cf9f25b36837efb2792070f362650b01f553d9d01bef080ebfefabR82)?

It shouldn't have been part of _construct_pypath in the first place, which was my mistake while refactoring. The root level BUILD.bazel file is only needed by pip_install. pip_parse already creates a root level BUILD.bazel via the extract_wheel function.

Reply:
Done in https://github.com/bazelbuild/rules_python/pull/457/commits/6e922451e38acef7e80105df20b2f9ffb4dd47eb.

The `here` link above links back to the comment, but I'm assuming you were referring to this `else`:

https://github.com/bazelbuild/rules_python/blob/master/python/pip_install/pip_repository.bzl#L82

I put it outside that conditional, since we do want the empty build file inside the repository where the `requirements.bzl` is generated-otherwise user code can't load from it. But it will be left out of the repositories for the individual packages still because it's not being injected here:

https://github.com/bazelbuild/rules_python/blob/master/python/pip_install/pip_repository.bzl#L215

Reply:
Thank you! Yes you read my mind. I was mixing up with whl_library invocation for that else block.